acceptance/suites/tests/020-external-ca/020_retrieve_new_cert.rb

```diff
@@ -0,0 +1,21 @@
+test_name 'QA-1393 - C62572 - Register Agent with Puppet Master using an External CA'
+
+lockfile = '/opt/puppetlabs/puppet/cache/state/agent_catalog_run.lock'
+lockcopy = '/tmp/foo'
```

Comment:
Line 17 is using `lockfile` - I don't see any other usages of `lockcopy` outside of teardown